<?xml version="1.0" encoding="UTF-8"?>
<launch>

	<param name="max_speed" value="1.0"/>
	<param name="min_speed" value="0.4"/>
	<param name="max_angle" value="1.0"/>
	<param name="tofs_default_max_dist"  value="10.0"/>
	<param name="min_dist"  value="0.4"/>
	<param name="topic_folder"  value=""/>
	<param name="left_is_green" value="False"/>
	

	<!-- Run nodes -->
	<node pkg="course_2023_pkg" type="pub_tofs2dist.py" name="pub_tofs2dist"/>

	<node pkg="course_2023_pkg" type="nav_tofs.py" name="nav_tofs" output="screen">

	</node>

    <!--<node pkg="course_2023_pkg" type="plot_sensors.py" name="plot_sensors"/>-->

</launch>
